src/App/Fossa/Analyze/ScanSummary.hs

```diff
@@ -113,6 +119,14 @@ instance Pretty ScanCount where
       , pretty numWarnings <> " analysis " <> plural "warning" "warnings" numWarnings
       ]
 
+staticOnlyAnalysisMessage :: Doc a
+staticOnlyAnalysisMessage =
+  vcat
+    [ "Performed static-only analysis!!!"
+    , "Results may be different in comparison to an analysis using build tools."
+    , "See " <> pretty staticAndDynamicStrategies <> " for more information."
```

Comment:
[nit] Due to the length of GitHub URLs, prefer to place it at the end:
```suggestion
    , "More information: " <> pretty staticAndDynamicStrategies
```
This way if the terminal wraps the URL the user isn't having to deal with that while parsing the sentence.